test/Altinn.AccessManagement.Tests/Controllers/Altinn2RightsControllerTest.cs

```diff
@@ -114,10 +116,90 @@ public async Task GetReceviedDelegations_ReturnOk(string header, string value, A
             string.Empty, "50005545", Assertions(
                 AssertFromContains(AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute, "50002203"),
                 AssertToContains(AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute, "50005545"),
-                AssertStatusCode(StatusCodes.Status200OK))
+                AssertStatusCode(HttpStatusCode.OK))
         },
     };
 
+    /// <summary>
+    /// Tests <see cref="RightsInternalController.ClearAccessCache(int, BaseAttributeExternal, System.Threading.CancellationToken)"/>
+    /// </summary>
+    [Theory]
+    [MemberData(nameof(ClearAccessCache_ReturnOk_input))]
+    public async Task ClearAccessCache_ReturnOk(string authnUserToken, int party, BaseAttributeExternal toAttribute, Action<HttpResponseMessage> assert)
+    {
+        var client = NewClient(NewServiceCollection(WithServiceMoq), [WithClientRoute("accessmanagement/api/v1/")]);
```

Comment:
[WithClientRoute("accessmanagement/api/v1/")]

do you need the brackets here?

Reply:
Nope, you are correct my good sir. I shalt have them removed